Modules/Test/classes/class.ilTestEvaluationGUI.php

```diff
@@ -1008,6 +1008,7 @@ public function outParticipantsPassDetails()
         $template->setVariable("PASS_DETAILS", $this->ctrl->getHTML($overviewTableGUI));
 
         $data = &$this->object->getCompleteEvaluationData();
+        $data->getParticipant($active_id)->setPassScoring($pass);
```

Comment:
For clarity:

The issue in release_7 is escalated in Modules/Test/classes/class.ilTestEvaluationUserData.php, specifically in lines 378 - 385:

```php
public function getScoredPass()
{
    if ($this->getPassScoring() == 1) {
        return $this->getBestPass();
    } else {
        return $this->getLastPass();
    }
}
```

Currently, this method always returns the result from `getLastPass` because `getPassScoring` is not set earlier.

This change addresses this issue and should resolve it accordingly.